src/utilities/include/par.h

```diff
@@ -87,9 +84,9 @@ inline constexpr ExecutionPolicy autoPolicy(int size) {
     return thrust::NAME(thrust::cpp::par, args...);                 \
   }
 
-#if MANIFOLD_PAR == 'T' && TBB_INTERFACE_VERSION >= 10000 && \
-    __has_include(<pstl/glue_execution_defs.h>)
-// sometimes stl variant is faster
+#if MANIFOLD_PAR != 'T' || \
+    (TBB_INTERFACE_VERSION >= 10000 && __has_include(<pstl/glue_execution_defs.h>))
+#if MANIFOLD_PAR == 'T'
```

Comment:
What does this mean with two `#if` statements in a row?

Reply:
I was changing this so that we will use stl as well when we only use single thread